modules/volume/include/inviwo/volume/processors/volumeregionmap.h

```diff
@@ -0,0 +1,76 @@
+/*********************************************************************************
+ *
+ * Inviwo - Interactive Visualization Workshop
+ *
+ * Copyright (c) 2022 Inviwo Foundation
+ * All rights reserved.
+ *
+ * Redistribution and use in source and binary forms, with or without
+ * modification, are permitted provided that the following conditions are met:
+ *
+ * 1. Redistributions of source code must retain the above copyright notice, this
+ * list of conditions and the following disclaimer.
+ * 2. Redistributions in binary form must reproduce the above copyright notice,
+ * this list of conditions and the following disclaimer in the documentation
+ * and/or other materials provided with the distribution.
+ *
+ * THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS "AS IS" AND
+ * ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE IMPLIED
+ * WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE ARE
+ * DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT OWNER OR CONTRIBUTORS BE LIABLE FOR
+ * ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES
+ * (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES;
+ * LOSS OF USE, DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND
+ * ON ANY THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT
+ * (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE OF THIS
+ * SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+ *
+ *********************************************************************************/
+
+#pragma once
+
+#include <inviwo/volume/volumemoduledefine.h>
+#include <inviwo/core/processors/processor.h>
+#include <inviwo/core/properties/ordinalproperty.h>
+#include <inviwo/core/ports/volumeport.h>
+#include <inviwo/core/ports/imageport.h>
+#include <inviwo/dataframe/datastructures/dataframe.h>
+#include <inviwo/core/processors/poolprocessor.h>
+#include <optional>
+#include <inviwo/dataframe/properties/columnoptionproperty.h>
+
+namespace inviwo {
+
+/** \docpage{org.inviwo.VolumeRegionMap, Volume Region Map}
+ * ![](org.inviwo.VolumeRegionMap.png?classIdentifier=org.inviwo.VolumeRegionMap)
+ * Explanation of how to use the processor.
+ * Indices are remapped to values provided in CSV file. Requires two columns. 1st column is old
+ * indices, second contains values to be mapped to.
+ * ### Inports
+ *   * __inputVolume__ Input volume
+ *
+ * ### Outports
+ *   * __outputVolume__ Output volume
+ *
+ * ### Properties
+ *
+ */
+class IVW_MODULE_VOLUME_API VolumeRegionMap : public Processor {
```

Comment:
Consider changing the name as it may not have anything to do with regions. I guess region is meant to imply that it is is integer values? Might make sense still depending on what other Atlas/Region processors are called
`VolumeValueMapper`
`VolumeAtlasMapper`

Reply:
We use region for a couple of other ones, so I think it makes an equal amount if sense.

Reply:
Still, it should either be VolumeRegionMapper or MapVolumeRegion in a similar fashion as we have VolumeRaycaster